main.go

```diff
@@ -26,21 +26,17 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 	_ "github.com/lib/pq"
 
+	"github.com/cloudspannerecosystem/harbourbridge/cmd"
 	"github.com/cloudspannerecosystem/harbourbridge/conversion"
 	"github.com/cloudspannerecosystem/harbourbridge/internal"
 	"github.com/cloudspannerecosystem/harbourbridge/web"
 )
 
 var (
-	badDataFile      = "dropped.txt"
```

Comment:
I think splitting up flags across different files/modules can make it hard to track them down.

Would like to chat about options for keeping them together. Having them in main is a good place, but accessing them from other modules becomes tricky (have to pass them as args, which gets messy). How about having a separate flags module?

The other option which we'd briefly discussed is having separate entry points for different ways to run HB e.g. for commandline vs ui. Let's discuss this a bit more.

Reply:
I agree that spreading flags across different modules is not a good idea. In fact, go recommendation is to have all command line flags being part of main package itself. For the purpose of this PR, using a separate flags submodule seems reasonable. Though, there are multiple ways of dealing with the issue here:

1. Collect related flags in structs such that passing around the flag values to cmd.CommandLine is less of an issue.

2. Define separate subcommands -- we need to think through all the current and future use cases when we jump on this.

Given these options, I agree that we should spend a little more time thinking about this before moving forward with this PR.

Reply:
I like your option 1 i.e. have a struct with all of the stuff we collect or compute from flags and environment variables. And then pass that to CommandLine. I think that's the cleanest thing to do for the moment. 
But let's not do it now, and add to TODO for the moment (see comment below).

Reply:
Done @https://github.com/cloudspannerecosystem/harbourbridge/pull/121/commits/96fafc8df69686a5aeac12ee5bec2c5656666d1e.